docs/src/dev-guide/coding-guidelines.md

```diff
@@ -3,8 +3,32 @@
 This project adheres to YScope's [contribution guidelines][yscope-guidelines] as well as the
 project-specific guidelines below.
 
+# Web Workers
+
+## Importing Web Workers
+
+When importing Web Worker files, use Vite's `?worker` query suffix syntax:
```

Comment:
I don't think "Web Workers" needs to be treated as a proper noun. I know MDN does so, but I can't see why they would. So can we capitalize is like normal words instead?

Reply:
Right, even in the specs https://html.spec.whatwg.org/multipage/workers.html they don't write the words in Title Case. let me update the docs